crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -336,6 +336,16 @@ when Mennas remained silent they cut out his tongue.
 Whether or not this angel is that same Mennas is something of a mystery, as he
 seems to have difficulty speaking.
 %%%%
+Mlioglotl
+
+A demon prince once betrayed his realm through a dark ritual to Lugonu.
+Becoming the twisted mass of corruption known as Mlioglotl was his
+reward.
```

Comment:
Fun :)

Reply:
Thanks! I was thinking it might be fun to do something in future like Maggie/Margery so that players can run into a scheming demon prince who's looking for a good place to conduct a dark ritual i.e. pre-corruption mlioglotl, as a mutually exclusive unique. No actual design for that in mind yet though.